src/Components/CollectionCard/CollectionCardModal.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Bin from '../../assets/icons/Bin.png';
+import Tag from '../../assets/icons/price-tag 1.png';
+import Star from '../../assets/icons/Star1.png';
+import Close from '../../assets/icons/Close.png';
+import Img from '../../assets/icons/image 22.png';
+
+export default function CollectionCardModal({ card, type }) {
```

Comment:
here the input should be just the card
import the collection constant and find the type using the card.type.id